src/components/SideNavbar/SideNavbar.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React, { useState } from 'react';
+import styled from 'styled-components';
+// eslint-disable-next-line sort-imports
+import SidenavMenu from './SidenavMenu';
+
+const Nav = styled.div`
+  width: 80px;
+  height: 1110px;
+  padding: 40px 24px 91px;
+
+  box-shadow: 0 0 5px 0 rgba(0, 0, 0, 0.15);
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const KeywordBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  margin: 20px 0;
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const HistoryBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   289px 0 0 0 ` : `margin: 20px 0`;
+  }};
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const ChatBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   45px 0 0 0 ` : `margin: 20px 0`;
+  }};
+  object-fit: contain;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const SetBlock = styled.div`
+  width: 32px;
+  height: 32px;
+  ${({ sideMenu }) => {
+    return sideMenu ? `margin :   397px 0 0 0 ` : `margin: 20px 0`;
+  }};
```

Comment:
본인이 생각해도 변수명이 이상하지 않나요?
sideMenu 는 명사인데 boolean 처럼 사용되고 있습니다.

```suggestion
  ${({ isSideMenu }) => {
    return isSideMenu ? `margin :   397px 0 0 0 ` : `margin: 20px 0`;
  }};
```

Reply:
092ef69

Reply:
변수명 수정했습니다(완)